### ISSUE TITLE
Feature: Pictures

### ISSUE THREAD

**Issue by laskinner** [laskinner/geodyssay-labs-landing#2]
# Todo

- [x] Create assets/images
- [x] Add images to folder
- [x] Add to HTML

# Styling

- [x] Rounded corners

Waiting on #1 to be finished first.

**Comment by laskinner** [comment]
Rouding corners in another ticket.
